crates/context_aware_config/migrations/2024-05-06-133756_custom_types/up.sql

```diff
@@ -0,0 +1,49 @@
+-- Your SQL goes here
+CREATE TABLE jsonschema_types (
+    id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+    type_name TEXT NOT NULL,
+    display_name TEXT NOT NULL,
+    type_schema JSON NOT NULL,
+    created_by TEXT NOT NULL,
+    created_at TIMESTAMP NOT NULL,
```

Comment:
@Datron , sorry for the late comment on this, but can we make this timestamp without timezone